.github/workflows/build.yml

```diff
@@ -1,8 +1,6 @@
 name: Create Release
 on:
   push:
-    branches:
-      - master
```

Comment:
GitHub Actions' syntax does not allow for actions to be triggered when tags are pushed to a specific branch; only when a tag is pushed anywhere or, separately, a commit is pushed to a specific branch.

The previous syntax caused git-release to trigger upon a tag update or new tag creation (good), and when any commit is pushed to master (bad, since git-release fails due to being fed a branch, which it cannot parse.)

See the GitHub Actions docs on branch/tag triggers: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onpushpull_requestbranchestags